### PR TITLE
Thu/degree programs

### PR DESCRIPTION
**Pull request by thumn**
## User Degrees and Multiple Programs
Users can select between degree programs "BFA" and "MFA." To allow for users to select multiple programs, the program enum was transitioned to a custom hash that is validated on the backend and reformatted and displayed in checkboxes on the Update Artist form on the front end. Artist information now scrolls to account for overflow.

### Related PRs
None.

### Migrations
20190415182531: Add degree to artists as enum.
20190419002135: Change program from enum to array of strings.
20190424090607: Make default value of degree 0: "BFA".

### Tests Performed, Edge Cases
Tested with submitting unchanged form.

### Screenshots
degree-programs

CC: @by-co

### CODE REVIEW

File: app/javascript/components/artists/UpdateArtist.jsx [hunk 1]
```diff
@@ -158,20 +175,42 @@ class UpdateArtist extends React.Component {
...
+                        <input
+                          onChange={() => this.toggleCheckbox(item)}
+                          type="checkbox"
+                          className="checkbox"
+                          value={convertSnakeCase(item)}
+                          id={`checkbox-${item}`}
+                          name="program"
+                          checked={(this.state.artist['program'].includes(item)) ? true : false}
+                        />
+                        <span className="filter-item">
```
Comment (by-co) [inline app/javascript/components/artists/UpdateArtist.jsx]: We can remove `filter-item`, it just constraints the width here which isn't necessary for this page

File: app/javascript/components/artists/UpdateArtist.jsx [hunk 2]
```diff
@@ -158,20 +175,42 @@ class UpdateArtist extends React.Component {
...
-              Object.keys(this.state.categories.program).map((obj, i) => {
+              Object.keys(this.state.categories.degree).map((obj, i) => {
                 return <option key={i} value={obj}>{convertSnakeCase(obj)}</option>
               })
             }
           </select>
+          <h5>Program</h5>
+
+          <div className="checkbox-container">
+                  {Object.keys(this.state.categories.program).map(item => (
```
Comment (by-co) [inline app/javascript/components/artists/UpdateArtist.jsx]: Nit: let's fix this indentation ⬅️

File: app/javascript/components/artists/UpdateArtist.jsx [hunk 3]
```diff
@@ -158,20 +175,42 @@ class UpdateArtist extends React.Component {
...
+                    <div className="mb2 checkbox-item" key={item}>
+                      <label className="ttc dib flex" htmlFor={`checkbox-${item}`}>
+                        <input
+                          onChange={() => this.toggleCheckbox(item)}
+                          type="checkbox"
+                          className="checkbox"
```
Comment (by-co) [inline app/javascript/components/artists/UpdateArtist.jsx]: Make it 📘 🔷 🚙 with
```suggestion
                          className="checkbox-denim"
```

**Comment by by-co** [comment]
Also make sure everything looks fine on the All Artists page :)
